docs/content/tracks/auto/bed_options_text.txt

```diff
@@ -5,17 +5,18 @@ title =
 height = 2
 # if you want to plot the track upside-down:
 # orientation = inverted
-# if you want to plot the track on top of the previous track. Options are 'yes' or 'share-y'. For the 'share-y'
-# option the y axis values is shared between this plot and the overlay plot. Otherwise, each plot use its own scale
+# if you want to plot the track on top of the previous track. Options are 'yes' or 'share-y'.
+# For the 'share-y' option the y axis values is shared between this plot and the overlay plot.
+# Otherwise, each plot use its own scale
 #overlay_previous = yes
 
 # If the bed file contains the exon
 # structure (bed 12) then this is plotted. Otherwise
 # a region **with direction** is plotted.
 # If the bed file contains a column for color (column 9), then this color can be used by
 # setting:
-# color = bed_rgb
-#if color is a valid colormap name (like RbBlGn), then the score is mapped
+#color = bed_rgb
```

Comment:
space or no space?

Reply:
I put space for explanations and no space when it is something that can be used.

Reply:
:+1: